pkg/skaffold/docker/parse.go

```diff
@@ -95,8 +95,35 @@ func readDockerfile(workspace, dockerfilePath string) ([]string, error) {
 
 	dispatchInstructions(res)
 
+	expandedPaths := make(map[string]bool)
+	for p := range depMap {
+		path := filepath.Join(workspace, p)
+
+		if _, err := os.Stat(path); err == nil {
+			expandedPaths[p] = true
+			continue
+		}
+
+		files, err := filepath.Glob(path)
+		if err != nil {
+			return nil, errors.Wrap(err, "glob")
```

Comment:
nit: a bit more descriptive error message?

Reply:
oh yes, let me fix that